app/url/create-url/createUrlResponseBuilder.js

```diff
@@ -0,0 +1,9 @@
+require('dotenv').config({path: '../../../.env'});
```

Comment:
0_0
dotenv need to be configured in the root file (index.js)
this operation is single-use, then we have access to `process.env` in every place